mtiapko/lab1-BasicIO/include/common.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __COMMON_H__
+#define __COMMON_H__
+
+#define DEBUG
+
+#ifdef DEBUG
+#	define FCHECK(func) if ((func) > WARNING_FSTATUS) prog_abort()
+#else
+#	define FCHECK(func) func
+#endif
+
+typedef signed char        int8_t;
```

Comment:
Just include `<stdint.h>` or `<inttypes.h>`. There's no need in defining these types manually